docs/docs/cli-reference/package-init.md

```diff
@@ -1,17 +1,22 @@
-# kurtosis package init
-This command initializes the current directory to be a [Kurtosis package][package] by creating a [`kurtosis.yml` file][kurtosis-yml] with the given package name.
+---
+title: package init
+sidebar_label: package init
+slug: /package-init
+---
+
+The `package init` command converts the current directory into a [Kurtosis package][package] by generating a new [`kurtosis.yml`][kurtosis-yml] file using the given package name.
 
 ```
 Usage:
-  kurtosis package init [flags] package_name
+  kurtosis package init [flags] $PACKAGE_NAME
 ```
 
-The `package_name` argument is the [locator][locators]to the package, in the format `github.com/USER/REPO`.
+The mandatory `#PACKAGE_NAME` argument is the [locator][locators] to the package, in the format `github.com/USER/REPO`.
```

Comment:
@leeederek I've just released the latest changes based on the feedback from our last all-hands and now this argument is optional.

Reply:
@leoporoli - what happens when I do not pass in the package name?

Reply:
@leeederek it's automatically created with this value: `github.com/example-org/example-package`